app/helpers/application_helper.rb

```diff
@@ -11,4 +11,9 @@ def bulma_flash_mapping(message_type)
   def beta_user?
     session[:is_beta_user].present?
   end
+
+  def has_unread_notifications?(user)
+    return false unless user
+    user.notifications.unread.present?
```

Comment:
`.any?` unless we want to eager load notifs

Reply:
huh, exists? is supposedly the most performant? https://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html